packages/back-end/src/services/slack.ts

```diff
@@ -13,6 +13,7 @@ export function verifySlackRequestSignature(
   res: ServerResponse,
   buf: Buffer
 ) {
+  // eslint-disable-next-line no-console
```

Comment:
We can just delete this console.log. I think that was a temporary debugging statement that we forgot to remove.